templates/layout/base.html

```diff
@@ -32,25 +32,12 @@
         {{> components/common/body }}
         {{> components/common/footer }}
 
+        <script src="{{cdn '/assets/js/bundle.js'}}"></script>
         <script>
-            function startupApp(stencilBootstrap) {
-                stencilBootstrap("{{template_file}}", {{jsContext}}).load();
-            }
+            // Exported in app.js
+            window.stencilBootstrap("{{template_file}}", {{jsContext}}).load();
         </script>
 
-        {{#if in_production}}
-            <script src="{{cdn '/assets/js/bundle.js'}}"></script>
-            <script>
-                startupApp(stencilBootstrap);
-            </script>
-        {{else}}
-            <script src="{{cdn '/assets/jspm_packages/system.js'}}"></script>
-            <script src="{{cdn '/assets/config.js'}}"></script>
-            <script src="{{cdn '/assets/js/dependency-bundle.js'}}"></script>
-            <script>
-                System.import("js/app").then(function(){ startupApp(stencilBootstrap)});
```

Comment:
JSPM and SystemJS serve two distinct purposes. SystemJS is basically a client-side module loader polyfill. It'll just load whatever dependency tree you give it, deferring execution however you define. The tree could be every file (like the old JSPM build) or modularized bundles (common, app-specific, etc.). Removing it here should be fine, though, since themes are simple enough that dependency sharing and module-specific cache busting probably aren't necessary.